docs/index.rst

```diff
@@ -27,19 +27,21 @@ ensure that Paramiko's dependencies build correctly:
 
 On Fedora, you can install these with the following command:
 
-For python2.x::
+Recommendation: Create virtual python environment for the following setup.
+Create virtual environment for python 3.x::
+$ python3 -m venv /path/to/new/virtual/environment
```

Comment:
Actually, according to my internal docs that I wrote some time ago, you do not specify a path, just the name you want to use for the venv. Here is what I wrote: 

You can create a virtual environment using the 'venv' command : 

~]$ cd  # this to ensure your are in home directory

~]$ python3 -m venv robottelo-master  # use names for your virtual environments that help you

~]$ cd robottelo

robottelo]$ source robottelo-test/bin/activate  # to enter a session
(robottelo-master) robottelo]$ deactivate # to end your session

robottelo]

Reply:
@swadeley The virtual environment does not need to be in robottelo directory. It can be created somewhere else too. This will give option to user to have this environment anywhere they want.

Reply:
> It can be created somewhere else too. 

thats what I was alluding to by "you do not specify a path"